### PR TITLE
Show all related organisations of a central worship service

### PR DESCRIPTION
**Pull request by nbittich**

### CODE REVIEW

File: app/routes/administrative-units/administrative-unit/core-data.js [hunk 1]
```diff
@@ -24,7 +23,7 @@ export default class AdministrativeUnitsAdministrativeUnitCoreDataRoute extends
           'primary-site.contacts',
           'is-sub-organization-of',
           'is-associated-with',
-          'sub-organizations',
+          //'sub-organizations',
```
Comment (Windvis) [inline app/routes/administrative-units/administrative-unit/core-data.js:26]: I would remove it if it's not actually needed anymore.

File: app/routes/administrative-units/administrative-unit/core-data/index.js [hunk 2]
```diff
@@ -2,10 +2,16 @@ import Route from '@ember/routing/route';
 
 export default class AdministrativeUnitsAdministrativeUnitCoreDataIndexRoute extends Route {
   model() {
+    const { id } = this.paramsFor('administrative-units.administrative-unit');
     return {
       administrativeUnit: this.modelFor(
         'administrative-units.administrative-unit.core-data'
       ),
+      subOrganizations: this.store.query('organization', {
```
Comment (Windvis) [inline app/routes/administrative-units/administrative-unit/core-data/index.js:10]: I would await this call. At the moment the page will be rendered before the data is loaded.

An alternative is to use a concurrency task and show a loading spinner while the data is loading. Similar to the organizations setup.

**Review by Windvis — CHANGES_REQUESTED** [review]

**Review by Windvis — APPROVED** [review]
Nice!

**Comment by nbittich** [comment]
wait before merging, not sure it works XD

**Comment by nbittich** [comment]
now it should be good, missing return in the task

**Pull request merged**